src/components/selectable/selectable.tsx

```diff
@@ -170,6 +170,11 @@ export type EuiSelectableProps<T = {}> = CommonProps &
      * Default: false
      */
     isPreFiltered?: boolean;
+    /**
+     * Screen readers will announce the selectableScreenReaderText after they
+     * read the `EuiSelectable` label and a brief pause
+     */
```

Comment:
minor documentation/copy nit - I don't think we need to re-state the name of the prop that we're describing.

I also think we need to make it clear that you're overriding the default selectable instructions, which consumers may not know. For example, the popover screen reader text is **in addition** to EUI's SR text, not **in replacement.**

```suggestion
    /**
     * Overrides `EuiSelectable`'s existing screen reader instructions that are announced on focus/interaction.
     * Screen readers will announce this text after they read the `EuiSelectable` label and a brief pause.
     */
```

Reply:
Actually on second thought, scratch the above suggestion. After our discussion on Slack, I would rather explicitly make the behavior of this prop to render **alongside**/before the existing default SR instructions rather than in place of:

```suggestion
    /**
     * Optional screen reader instructions to announce upon focus/interaction. This text is read out
     * after the `EuiSelectable` label and a brief pause, but before the default keyboard instructions for
     * interacting with a selectable are read out.
     */
```